test/integration/fixtures/blocks/core__quote__deprecated-2.serialized.html

```diff
@@ -1,5 +1,5 @@
 <!-- wp:quote {"align":"right"} -->
```

Comment:
This doesn't seem right.  The final html should have the new attribute..

Reply:
There is something wrong with the deprecation still.  @andrewserong suggested (above) that the fixture files should be unchanged.
I have not manually changed them, I ran the command `npm fixtures:regenerate` because of the reported test failure.

Reply:
> There is something wrong with the deprecation still.

Yes, that seems like it might be the case.

>  @andrewserong suggested (above) that the fixture files should be unchanged.

I think I might have been wrong about that, apologies, I think I might have missed some nuance in earlier reviews. In principle the block markup of this fixture should be unchanged (that is, we'd expect the HTML to be `<blockquote class="wp-block-quote has-text-align-right">`), but we'd also expect the block attributes in the block comment to have a value for `textAlign` in the serialized version.

Reply:
Who is the best person to ping to review the deprecation?